src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,30 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { getUser } from '../../api';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+
+interface Props {
+  todo: Todo;
+  setSelectedTodoId: React.Dispatch<React.SetStateAction<number>>;
+}
+
+export const TodoModal: React.FC<Props> = ({ todo, setSelectedTodoId }) => {
```

Comment:
Event handler functions should be named using the handleElementEvent format. In your case, the function passed to setSelectedTodoId could be named handleSetSelectedTodoId. This will make your code more readable and maintainable. This is a code style issue.